src/routes/_layout.svelte

```diff
@@ -1,29 +1,23 @@
 <script lang="ts">
   export let segment = undefined;
```

Comment:
This line seems to be triggering warnings for me locally (running `npm run dev`):

> Header_1 has unused export property 'segment'. If it is for external reference only, please consider using `export const segment`

Reply:
Tracked down the warning. It looks like using `segment` in this way has been deprecated, and there is a `page` store that takes its place. The `segment` and its replacement are used for getting access to  parts of the URL.

Will remove it and look into replacing it with the `page` store.